finder-app/Makefile

```diff
@@ -0,0 +1,33 @@
+# Makfile to build the writer application
+# Usage for Cross compiling:$ make writer CROSS_COMPILE=<cross_compiler_version>
+
+#native compiler
+CC = gcc
+
+#compiler flags
+CFLAGS = -g -Wall -Werror
+
+#source files
+SRCS = writer.c
+
+#object files
+OBJS = $(SRCS:.c=.o)
+
+#Executable Target file
+TARGET = writer
+
+
+%.o: %.c
+	$(CROSS_COMPILE)-$(CC) $(CFLAGS) -c -o $@ $<
+
+all: $(TARGET)
+
+$(TARGET): $(OBJS)
+	$(CROSS_COMPILE)-$(CC) $(CFLAGS) $^ -o $@
+
+
+.PHONY: clean
+
+clean:
+	$(RM) $(TARGET) *.o *.elf *.map
```

Comment:
@ajsanthosh14 what is $(RM) here?

Reply:
@swatikadivar RM is a make variable/macro which is equivalent to rm -f

Reply:
you can definitely use it but you haven't defined it anywhere.

Reply:
It's an inbuilt macro, we don't need to define it right? When I do make clean, it does the work and removes all the files mentioned.

Reply:
i just read about it. it works. i'll update it. good to know. :)

Reply:
Awesome, no worries.
Thank you.